src/__tests__/fixtures/regression.css

```diff
@@ -0,0 +1,7 @@
+@custom-selector :--button .button;
+
+:--button {
+  &--active {
```

Comment:
@tabatkins let's remove for a moment the `--wat`. How should be the output of this
```
@custom-selector :--blah .blah p;

:--blah {
  & span { ... }
}
```